src/components/ebay-avatar/index.marko

```diff
@@ -32,13 +18,14 @@ $ const {
     aria-label=a11yText
     class=[
         "avatar",
+        state.imagePlacement === "fit" && "avatar--fit",
         inputClass,
         size && `avatar--${size}`,
-        `avatar--${getColorForText(username, color)}`,
+        color && !img && `avatar--${getColorForText(username, color)}`,
```

Comment:
@agliga doesn't checking for `color` here defeat the purpose of `getColorForText`? Looks like `color` is used as the default, so this is the same as `` !img && `avatar--${color}` ``

Reply:
We removed the default because of the image scrim.

Reply:
Okay, in that case we don't need `getColorForText` because this is the same as `` color && !img && `avatar--${color}` ``